### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Microsoft IIS Home Page_Zooming: when browser screen is zoomed to 200%, some contents and controls are not visible properly.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#82]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated:** MAS1.4.4

**Repro Steps:**
1. Open "Microsoft IIS" Home Page using the URL: https://manage.iis.net/
2. Set browser zooming to 200% and verify that all the contents and controls are visible properly.

**Actual Result:**
When browser zooming is set to 200%, "Microsoft IIS" link , "Skip This" button and welcome message are not visible properly.

**Expected Result:**
All the contents and controls of the page should be visible clearly even if the browser screen is zoomed to 200%.

**User Impact:**
This Success Criterion helps people with low vision by letting them increase text size in content so that they can read it.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}

**Attachment:**
![bug 82_zooming issue](https://user-images.githubusercontent.com/38521133/44582973-04736c00-a7c1-11e8-974d-d34472d17ccf.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Nikhitha-13** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Resize; #MAS1.4.4; #C+E-H; #.NETWebsites; #Chrome; #Edge

**Comment by fsteffi** [comment]
#A11ySev3;
